pkg/cmd/authTokens.go

```diff
@@ -76,3 +77,21 @@ func (cmd *AuthTokensCommand) createAuthTokensCobraCmd(
 
 	return authTokensCmd, err
 }
+
+func addLoginIdFlagToAuthTokensGet(cmd *cobra.Command, isMandatory bool, authTokensGetCmdValues *AuthTokensCmdValues) {
+
+	flagName := "id"
```

Comment:
function is called addLoginIdFlag... but the flag is `id`? Shouldn't this be `loginid`?